script.js

```diff
@@ -0,0 +1,10 @@
+const nav = document.querySelector('.nav-items');
+const humburger = document.querySelector('.humburger');
```

Comment:
- [ ] [REQUIRED] Great implementation @MAHDIA001 , however, the page has a horizontal scroll bar that needs to be removed. This is causing the close button (X) to hide on the mobile screen. Kindly take a look at the image below.
- [ ] [REQUIRED] Kindly ensure that you maintain the same padding and margin in between the pop-up up options (Portfolio, About, Contact) as provided in the design template.
### Your Design
![Screenshot from 2022-09-06 17-46-21](https://user-images.githubusercontent.com/58692590/188694114-d80927ae-8f77-47b6-abcc-665c2dea7ea4.png)